composer.json

```diff
@@ -106,7 +106,8 @@
         "talkroute/message-segment-calculator": "^1.0",
         "tapp/filament-timezone-field": "^3.0",
         "tpetry/laravel-postgresql-enhanced": "^0.40",
-        "twilio/sdk": "^7.15"
+        "twilio/sdk": "^7.15",
+        "phpoffice/phpspreadsheet": "dev-security-patch as 1.29.1"
```

Comment:
This does not seem to be in the correct spot alphabetically is it?

If you had ran `composer require phpoffice/phpspreadsheet` first it would have put it in the correct spot with `*` as the version. Then you can change it to `dev-security-patch as 1.29.1` and run update.

Or just move this to the right spot, either way.

Reply:
Apologies ! Corrected the order in recent changes !